rotkehlchen/chain/ethereum/modules/airdrops/decoder.py

```diff
@@ -81,21 +95,25 @@ def __init__(
         self.convex = A_CVX.resolve_to_evm_token()
         self.fpis = A_FPIS.resolve_to_evm_token()
         self.elfi = A_ELFI.resolve_to_evm_token()
+        self.vcow = A_VCOW.resolve_to_evm_token()
 
     def _decode_uniswap_claim(self, context: DecoderContext) -> DecodingOutput:
         if context.tx_log.topics[0] != UNISWAP_TOKEN_CLAIMED:
             return DEFAULT_DECODING_OUTPUT
 
-        user_address = hex_or_bytes_to_address(context.tx_log.data[32:64])
+        hex_or_bytes_to_address(context.tx_log.data[32:64])
```

Comment:
this here looks suspicous. Is a no-op. I believe the user address should be this instead